### PR TITLE
BETA: Register prtksunal.is-a.dev

### PR DESCRIPTION
**Pull request by prateekmedia**
Added `prtksunal.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/prtksunal.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "prateekmedia",
+        "email": "prtksunal@gmail.com"
+    },
+    "record": {
+        "CNAME": "prateekmedia.github.io"
```
Comment (DEV-DIBSTER) [inline domains/prtksunal.json:7]: CNAME just redirects back to our domain. Please display content.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/prtksunal.json (hunk 1 above)
Comment (prateekmedia) [reply domains/prtksunal.json:7]: Done!

**Comment by prateekmedia** [comment]
@DEV-DIBSTER Please check

**Comment by prateekmedia** [comment]
My aim is to use this domain and show the content from my github.io subdomain, let me know if I am doing it right or not.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add content to your website.

**Review by wdhdev — APPROVED** [review]

**Pull request merged**